TundraTD/Assets/Scripts/Level/TickConvertation.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Level
+{
+    static class TickConvertation
+    {
+        public static int SecondToTick(float second) => (int)(second * 10);
```

Comment:
I suppose the extension method would be a good solution for this static class
```suggestion
        public static int SecondsToTicks(this float seconds) => (int)(second * 10);
```